src/workloads/execution/UnionWith.yml

```diff
@@ -10,12 +10,13 @@ Actors:
     Database: &db test
     Threads: 1
     CollectionCount: 20
-    DocumentCount: 10000
-    BatchSize: 1000
+    DocumentCount: 5000
+    # Choose a batchSize and document size such that all documents can fit in a single 16MB batch.
+    BatchSize: &batchSize 100000
```

Comment:
I've modified the documents to be roughly 100bytes, which with some quick math means that (20 collections x 5000 docs/collections * 100 doc size ~= 10MB). So I think we should be good to fit into a 16MB batch....does that make sense or did I forget to carry a 1 somewhere?

Reply:
Makes sense to me! Out of curiosity how did you check the generated document size?

Reply:
I went through each of the fields and estimated their size :)

I likely overestimated but it should be fine for this purpose.